src/audiences/ui/components/list-row.js

```diff
@@ -140,11 +140,17 @@ const ListRow = props => {
 				</div>
 			</td>
 			<td>
-				<StatusToggle
-					disabled={ ! canEdit }
-					status={ post.status }
-					onChange={ onStatusChange }
-				/>
+				{ canEdit && (
+					<StatusToggle
+						status={ post.status }
+						onChange={ onStatusChange }
+					/>
+				) }
+				{ ! canEdit && post.status === 'publish' && (
+					<Fragment>
+						<span className="active">{ __( 'Active', 'altis-analytics' ) }</span>
+					</Fragment>
```

Comment:
The opposite status isn't displayed for Contributors because they don't have access to draft posts they didn't create, and they don't have access to create audiences so I left it out.